src/main/java/core/basesyntax/service/activitieswithfruitservice/impl/ActivitiesStrategyImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.activitieswithfruitservice.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activitieswithfruitservice.ActivitiesStrategy;
+
+public class ActivitiesStrategyImpl implements ActivitiesStrategy {
+    @Override
+    public void getActivitiesWithFruit(String code, FruitTransaction fruitTransaction) {
```

Comment:
1) it's not good to name method with `get` that is return nothing
2) Why do we need code separately if fruitTransaction contains it as field?
```suggestion
    public void performTransaction(FruitTransaction fruitTransaction) {
```

Reply:
ok thx